components/form/Input.js

```diff
@@ -1,16 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Input = ({ className, Icon, label, name, placeholder, type }) => {
+const Input = ({ className, Icon, label, name, placeholder, type, register }) => {
   return (
```

Comment:
We can pass the error props (like register) into the Input component and check if the current field has an error. Here is the implementation we did today:

1. Pass the error as a prop on line 4

2. Check if the field currently has an error:
```js
  const hasErrors = !!errors?.[name];
```

3. If it has an error, add the `border-red-500` border (you can use a different className) to the Input className

```js
hasErrors ? 'border-red-500' : ''
```

4. Check if the error exist and add it to the Input component:

```js
  {errors?.[name] && <div className="text-amcovad-danger">{errors?.[name]?.message}</div>}
```